x-pack/plugins/ml/server/models/data_frame_analytics/analytics_manager.ts

```diff
@@ -47,8 +47,8 @@ export class AnalyticsManager {
 
   private async initData() {
     const [models, jobs] = await Promise.all([
-      this._mlClient.getTrainedModels(),
-      this._mlClient.getDataFrameAnalytics({ size: 1000 }),
+      this._mlClient.getTrainedModels({ size: 10000 }),
```

Comment:
for consistency this the file below, this file could also use a local `DEFAULT_SIZE` variable.